lib/PuppeteerSharp/Page.cs

```diff
@@ -369,6 +367,11 @@ public int DefaultTimeout
         /// </summary>
         public Accessibility Accessibility { get; }
 
+        /// <summary>
+        /// `true` if drag events are being intercepted, `false` otherwise.
+        /// </summary>
+        public bool IsDragInterceptionEnabled { get; internal set; }
```

Comment:
I recommend making the setter `private set;`

Reply:
done!